libr/include/r_util/r_annotated_code.h

```diff
@@ -0,0 +1,95 @@
+
+#ifndef R2GHIDRA_ANNOTATEDCODE_H
+#define R2GHIDRA_ANNOTATEDCODE_H
+
+#include <r_core.h>
+#include <r_types.h>
+#include <r_vector.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum r_syntax_highlight_type_t {
+	R_SYNTAX_HIGHLIGHT_TYPE_KEYWORD,
+	R_SYNTAX_HIGHLIGHT_TYPE_COMMENT,
+	R_SYNTAX_HIGHLIGHT_TYPE_DATATYPE,
+	R_SYNTAX_HIGHLIGHT_TYPE_FUNCTION_NAME,
+	R_SYNTAX_HIGHLIGHT_TYPE_FUNCTION_PARAMETER,
+	R_SYNTAX_HIGHLIGHT_TYPE_LOCAL_VARIABLE,
+	R_SYNTAX_HIGHLIGHT_TYPE_CONSTANT_VARIABLE,
+	R_SYNTAX_HIGHLIGHT_TYPE_GLOBAL_VARIABLE,
+} RSyntaxHighlightType;
+
+typedef enum r_code_annotation_type_t {
+	R_CODE_ANNOTATION_TYPE_OFFSET,
+	R_CODE_ANNOTATION_TYPE_SYNTAX_HIGHLIGHT,
+	// ...
+} RCodeAnnotationType;
+
+typedef struct r_code_annotation_t {
+	size_t start;
+	size_t end;
+	RCodeAnnotationType type;
+	union
+	{
```

Comment:
Should be `union {`, as well as `struct {` below. This was wrongly formatted in r2ghidra.